src/style/reset.css

```diff
@@ -0,0 +1,4 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
it's better not to add `margin: 0` for all elements. leave it only for needed selectors